decide/lic6.test.cpp

```diff
@@ -0,0 +1,53 @@
+#include <cassert>
+#include <cmath>
+#include <cstdlib>
+#include <decide/decide.hpp>
+#include <decide/lics.hpp>
+
+double distanceFromLine(Coordinate &point, Coordinate &line_start,
+                        Coordinate &line_end);
+
+// The point is closest to the line
+void test_distance_from_line() {
+  auto point = Coordinate(-2, 5);
+  auto start = Coordinate(100, -73.75);
+  auto end = Coordinate(-100, 76.25);
+  double result = distanceFromLine(point, start, end);
+  assert(abs(result - 9.0 / 5.0) < 0.0001);
```

Comment:
Is the result supposed to be 9/5? If so, looks good.

Reply:
It is! I used an example from some math website with these exact coordinates.